com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/MotionVectorVertexShaderCommon.hlsl

```diff
@@ -107,15 +106,21 @@ PackedVaryingsType MotionVectorVS(inout VaryingsType varyingsType, AttributesMes
     {
         bool hasDeformation = unity_MotionVectorsParams.x > 0.0; // Skin or morph target
 
-        float3 effectivePositionOS = (hasDeformation ? inputPass.previousPositionOS : inputMesh.positionOS);
+        float3 deformedPrevPos = inputPass.previousPositionOS;
+
+#if defined(DOTS_INSTANCING_ON)
+        FetchComputeVertexPosition(inputMesh.positionOS, deformedPrevPos, inputMesh.vertexID);
```

Comment:
Question: is dots skinning cumulative with regular skinning? 

inputPass.previousPositionOS contain the previous skinned mesh if any, so I am curious.

I have the feeling that we should rather have a code like this:

#if defined(DOTS_INSTANCING_ON)
bool hasDeformation = asint(unity_DOTSDeformationParams).z; // DOTS skinning
if (hasDeformation)
     FetchComputeVertexPosition(inputMesh.positionOS, inputPass.previousPositionOS, inputMesh.vertexID);
#else
bool hasDeformation = unity_MotionVectorsParams.x > 0.0; // Skin or morph target
float3 deformedPrevPos  = inputPass.previousPositionOS);
#endif

float3 effectivePositionOS = (hasDeformation ? deformedPrevPos  : inputMesh.positionOS);

Thought?